lists/mx-cmx_cpa.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": {
+    "en": "Budget Classification of Public Entities (Mexico City)",
+    "local": "Entes Públicos de la Ciudad de México"
+  },
+  "url": "https://datos.cdmx.gob.mx/dataset/entes-publicos-de-la-ciudad-de-mexico",
+  "description": {
+    "en": "Catalog of dependencies and municipalities that are registered as entities that receive a budget according to the budget for expenditures for the current year.\n\nThis list provides Mexico City's administrative classification, with the same meanings as [MX-CPA](https://org-id.guide/list/MX-CPA) but at the level of Mexico City."
+  },
+  "coverage": [
+    "MX"
+  ],
+  "subnationalCoverage": [
+    "MX-DIF"
```

Comment:
@mrshll1001 - I think MX-DIF was replaced by MX-CMX, however our org-id codelist didn't keep up. See https://www.iso.org/obp/ui/#iso:code:3166:MX

I don't want that bug to block merging of this list, though. I'll file a separate issue so that we can mop this up later.

Reply:
Separate issue filed: https://github.com/org-id/register/issues/477